chart/templates/configmap.yaml

```diff
@@ -16,6 +16,8 @@ data:
       metricsSink: {{ .Values.controller.metricsSink | quote }}
       deleteOnly: {{ .Values.controller.deleteOnly }}
       imagePullSecrets: {{ .Values.images.pullSecrets }}
+      defaultDuration: {{ .Values.controller.defaultDuration }}
+      expiredDisruptionGCDelay: {{ .Values.controller.expiredDisruptionGCDelay }}
```

Comment:
I was curious about the need for this. Can't we just rely on the `duration` field and delete the CR once the time-window is exceeded?

Reply:
Personally, yes, I think that would work and would be fine, but concern has been expressed that users might find it surprising for their disruption CRDs to suddenly disappear. What do you think?

Reply:
Thanks for the context. Although it adds some complexity it sounds sensible :)